core/blockchain.go

```diff
@@ -1882,12 +1882,10 @@ func (bc *BlockChain) insertChain(chain types.Blocks, setHead bool) (int, error)
 		storageUpdateTimer.Update(statedb.StorageUpdates)               // Storage updates are complete(in validation)
 		accountHashTimer.Update(statedb.AccountHashes)                  // Account hashes are complete(in validation)
 		storageHashTimer.Update(statedb.StorageHashes)                  // Storage hashes are complete(in validation)
-		triehash := statedb.AccountHashes + statedb.StorageHashes       // The time spent on tries hashing
-		trieUpdate := statedb.AccountUpdates + statedb.StorageUpdates   // The time spent on tries update
 		trieRead := statedb.SnapshotAccountReads + statedb.AccountReads // The time spent on account read
 		trieRead += statedb.SnapshotStorageReads + statedb.StorageReads // The time spent on storage read
-		blockExecutionTimer.Update(ptime - trieRead)                    // The time spent on EVM processing
-		blockValidationTimer.Update(vtime - (triehash + trieUpdate))    // The time spent on block validation
+		blockExecutionTimer.Update(ptime)                               // The time spent on EVM processing
+		blockValidationTimer.Update(vtime)                              // The time spent on block validation
```

Comment:
It seems that the definitions of `blockExecutionTimer` and `blockValidationTimer` have been re-defined. While I am not sure if this is intended. if yes, could you please explain why this change is better?

Reply:
Yes; this PR does not delete any metrics, but only adjusts the calculation method of metrics, because when analyzing performance bottlenecks during `--metrics.expensive`, blockExecutionTimer/blockValidationTimer/blockWriteTimer may be negative when deducting concurrency latency. This is a bit confusing. In fact, if we pay attention to the metrics of internal concurrent actions, we can directly configure a corresponding panel, and  I think it is better to have clear semantics of blockExecutionTimer/blockValidationTimer/blockWriteTimer.   :)
This is just a slight change and ultimately depends on the reviewers' suggestions.

Reply:
Without `--metrics.expensive`, These `triehash/trieUpdate/..Commits` are all equal to 0.

Reply:
ACK. Thanks.